packages/nuxt/src/app/nuxt.ts

```diff
@@ -164,6 +165,12 @@ export interface Plugin<Injections extends Record<string, unknown> = Record<stri
 export interface ObjectPluginInput<Injections extends Record<string, unknown> = Record<string, unknown>> extends PluginMeta {
   hooks?: Partial<RuntimeNuxtHooks>
   setup?: Plugin<Injections>
+  /**
+   * Execute plugin in parallel with other parallel plugins.
+   *
+   * @default false
+   */
+  parallel?: boolean
```

Comment:
would it be clearer to change it to `blocking: false` (with default `true`)?

Reply:
blocking: false seems a good idea too as anothe feature 👍🏼  but i think parallel is still blocker just orderless.

Reply:
Yes, parallel would still need to block app loading, but not other plugins running... And I do think we probably could consider running plugins within custom order groups in parallel by default....